scripts/travis/codegen_verification.sh

```diff
@@ -42,7 +42,7 @@ function runGoFmt() {
 }
 
 function runGoLint() {
-    warningCount=$("$GOPATH"/bin/golint $(GO111MODULE=off go list ./... | grep -v /vendor/ | grep -v /test/e2e-go/) | wc -l | tr -d ' ')
+    warningCount=$("$GOPATH"/bin/golint $(GO111MODULE=on go list ./... | grep -v /vendor/ | grep -v /test/e2e-go/) | wc -l | tr -d ' ')
```

Comment:
At this point, can't we just omit the `GO111MODULE=on` ?

Reply:
I'll give it a try, it should work since we export it that way.